src/adapters/v00/marketplace_adapter.js

```diff
@@ -1,3 +1,5 @@
+const OFFER_STATUS = ['error','created', 'accepted', 'finalized', 'buyerReviewed']
```

Comment:
I think this is missing status "disputed" which is 3.
See https://github.com/OriginProtocol/origin-js/blob/master/contracts/contracts/marketplace/v01/Marketplace.sol#L55

And actually it looks like the contracts and this file are out of sync. In the code below, 'OfferFinalized' event should set rawOffer.status to '4' and 'OfferData' should set it to 5.

Out of scope for this PR but we should really write some unit tests to catch these type of things :)

Reply:
Yes, there appear to be three different ways the status comes back from three different methods in the adapter. :(